src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift

```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-class CachingHTTPClient: AppConfigurationFetching, StatisticsFetching {
+class CachingHTTPClient: AppConfigurationFetching, StatisticsFetching, QRCodePosterTemplateFetching {
```

Comment:
Could you please mark code that fulfills the protocol

Reply:
The marks were already there but I rename them to QRCodePosterTemplateFetching.